apps/artemis-web/components/hijack-info/hijack-info.tsx

```diff
@@ -104,7 +103,208 @@ class HijackInfoComponent extends Component<any, any> {
       return "all";
   }
 
-  render() {
+  getMobileInfo(mRef, hijackKey, commentRef, event_data, type): any {
```

Comment:
why this large change here?

Reply:
Because I was trying to pass the code smells :P

Reply:
does it change the logic, or is it simply a formatting change? hard to understand with a large PR :)

Reply:
no it is just method extraction. I moved the HTML to a new method